sotopia/generation_utils/generate.py

```diff
@@ -777,7 +765,7 @@ def process_history(
 async def agenerate_init_profile(
     model_name: str,
     basic_info: dict[str, str],
-    bad_output_process_model: str = DEFAULT_BAD_OUTPUT_PROCESS_MODEL,
```

Comment:
why remove the default?

Reply:
To use custom model without OpenAI keys, the original way doesn't work. Now we use the the model as the default reformat model, but allow the user to use other models for reformatting.

Reply:
would this break the existing running script that does not specify reformatting models? can we still have a default one?

Reply:
yeah, the default one would be the one used for generating the output for the first time.